src/app/organizations/organization-details/organization-details.component.html

```diff
@@ -9,17 +8,20 @@
                 <i class="fas fa-long-arrow-alt-left mr-3"></i>
                 {{ 'organizations.details.title.back' | translate }}
             </div>
-            <button [disabled]="!isWalletVerified(states.orgWallet)"
-                    class="btn"
-                    routerLink="create_project">
-                <i class="fas fa-plus mr-lg-3"></i>
-                <span class="btn__text">
+
+            <ng-container *ngIf="!isPublic && (orgWallet$ | async) as wallet">
```

Comment:
I'm not sure that it works.. but, in this case, `wallet` variable will be `boolean` because you are assigning the entire expression (`!isPublic && (orgWallet$ | async)`) to it. 
Wrap it in one more `ng-container` for `!isPublic` and leave just `(orgWallet$ | async)` here.